src/js/components/Clock/Clock.js

```diff
@@ -52,7 +52,7 @@ class Clock extends Component {
   static getDerivedStateFromProps(nextProps, prevState) {
     const { hourLimit, time } = nextProps;
     const { elements } = prevState;
-    if (!elements || time) {
```

Comment:
What if the caller isn't using `run` but changes the `time`? I think this means we won't recalculate the elements.

Reply:
@ericsoderberghp good question, I added a story called `Explicit Time` where you can see that there is no impact on recalculation of time whether the time is being set or not. 
Also, the snapshots haven't been changed on tests that are using explicit time.

In general, I think that if this approach of fix is considered, lines #55 & #57 has the same condition, and I'll need to do further clean-up. but I want to see first if you have other inputs for me to consider.

Reply:
I'm not concerned about the initial setting of `time`, I'm more concerned if the `time` value is changed. For instance, if I have a details panel that renders a Clock based on what has been selected in a list. So, the same `<Clock />` but the `time={value}` has a changing value.

Reply:
gotcha, that is a good test scenario. I'll test that as well.